rails_translation_manager.gemspec

```diff
@@ -24,5 +24,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "minitest"
-  spec.add_development_dependency "rspec"
+  spec.add_development_dependency 'rspec'
```

Comment:
I think this came out of a rebase that went wrong, as Peter's PR originally used single quotes and then changed to double quotes for consistency with the rest of the file. I see you switch back to double quotes in the next commit. Could you fix this up here too so that there's no change overall?